tools/run_tests/sanity/sanity_tests.yaml

```diff
@@ -24,6 +24,7 @@
 - script: tools/distrib/check_pytype.sh
 - script: tools/distrib/clang_format_code.sh
 - script: tools/distrib/clang_tidy_code.sh
+  cpu_cost: 1000
```

Comment:
What does this do?

Reply:
This is to make clang-tidy run exclusively when running on Kokoro machine. This is required to run bazel without being interrupted by other tasks.

Reply:
Maybe add a comment here explaining this?

Reply:
Good idea. This is a yaml file so I should leave comments.